README.md

```diff
@@ -12,13 +12,16 @@ Currently, the module contains the following shapes
 * **Stack**: three stack shapes (like ingots or bars; 1/3, 2/3 and Full Stack)
 * **Barrel**: barrel shape and texture
 * **Connectors**: connectors for cables, axles, ... (two sizes, different variants)
+* **TimberFrame**: cross, diagonal, horizontal, vertical, plain
```

Comment:
Could we have a screenshot showing the new blocks in action? Maybe just take one the HjerlHede structures later on once the templates are corrected.